Submitted By: Mario Fetka (mario dot fetka at gmail dot com)
Date: 2008-08-25
Initial Package Version: 2.0 pre
Origin: me
Upstream Status: unknown
Description: correct ffmpeg location
 
diff -Naur k3b-2.0_pre20080808.orig/plugins/decoder/ffmpeg/k3bffmpegdecoder.cpp k3b-2.0_pre20080808/plugins/decoder/ffmpeg/k3bffmpegdecoder.cpp
--- k3b-2.0_pre20080808.orig/plugins/decoder/ffmpeg/k3bffmpegdecoder.cpp	2008-08-25 11:51:38.000000000 +0000
+++ k3b-2.0_pre20080808/plugins/decoder/ffmpeg/k3bffmpegdecoder.cpp	2008-08-25 11:52:00.000000000 +0000
@@ -21,8 +21,8 @@
 #include <kdebug.h>
 
 extern "C" {
-#include <ffmpeg/avcodec.h>
-//#include <libavcodec/avcodec.h>
+// #include <ffmpeg/avcodec.h>
+#include <libavcodec/avcodec.h>
 }
 
 #include <math.h>
diff -Naur k3b-2.0_pre20080808.orig/plugins/decoder/ffmpeg/k3bffmpegwrapper.cpp k3b-2.0_pre20080808/plugins/decoder/ffmpeg/k3bffmpegwrapper.cpp
--- k3b-2.0_pre20080808.orig/plugins/decoder/ffmpeg/k3bffmpegwrapper.cpp	2008-08-25 11:51:38.000000000 +0000
+++ k3b-2.0_pre20080808/plugins/decoder/ffmpeg/k3bffmpegwrapper.cpp	2008-08-25 11:52:22.000000000 +0000
@@ -17,10 +17,10 @@
 #include <config-k3b.h>
 
 extern "C" {
-#include <ffmpeg/avcodec.h>
-#include <ffmpeg/avformat.h>
-// #include <libavcodec/avcodec.h>
-// #include <libavformat/avformat.h>
+// #include <ffmpeg/avcodec.h>
+// #include <ffmpeg/avformat.h>
+#include <libavcodec/avcodec.h>
+#include <libavformat/avformat.h>
 }
 
 #include <string.h>
